scripts/vaccination_alerts/README.md

```diff
@@ -0,0 +1,93 @@
+# Vaccination Alerts
+
+Outline of the workflow:
+
+1. Generate the list of locations that need to be alerted
+2. Generate a list of (user email, location) to be alerted
+3. Send the emails
+
+## 1. generate-vaccine-alerts-info
+
+This script determines which locations have updated vaccination information by comparing the information in Firebase with the information from the CMS.
+
+### Firebase
+
+```yaml
+info:
+  vaccinationInfoUpdates:
+    '11':
+      emailAlertVersion: 0
+    '12':
+      emailAlertVersion: 0
+```
+
+### From the CMS
+
+```yaml
+'11':
+  emailAlertVersion: 3 # Updated
+  content: 'Phase A...'
+'12':
+  emailAlertVersion: 0
+  content: 'Phase 1...'
+'13':
+  emailAlertVersion: 3 # New
+  content: 'Phase 1...'
+```
+
+In this case, for example, we need to send emails for udpates on location "11" and location "13" (location "12" didn't have any updates).
+
+For locations that have updated information, we save a copy of the CMS information in the file `vaccination-alerts.json`. The information in `vaccination-alerts.json` will be used in later phases to determine the list of users to email and to render the emails.
+
+```yaml
+'11':
+  emailAlertVersion: 3
+  content: 'Phase A...'
+'13':
+  emailAlertVersion: 3
+  content: 'Phase 1...'
+```
+
+## 2. generate-users-to-email
+
+Using the locations in `vaccination-alerts.json`, we query email addresses subscribed to these locations from the the Firebase collection `alerts-subscriptions`.
+
+```yaml
+alerts-subscriptions:
+  pablo@covidactnow.org:
+    locations: ['11']
+  chelsi@covidactnow.org:
+    locations: ['11', '13']
+```
+
+Then, we update store the list of locations and email addresses to send in the Firebase collection `vaccination-alerts`, in the document `snapshotId`, so we can track the locations and users that we email each time.
+
+```yaml
+vaccination-alerts:
+  '1500':
+    locations:
+      '11':
+        emails:
+          pablo@covidactnow.org: { sentAt: null }
+          chelsi@covidactnow.org: { sentAt: null }
+      '13':
+        emails:
+          chelsi@covidactnow.org: { sentAt: null }
+```
+
+We update the `info` collection with the latest updated date.
```

Comment:
Not 100% sure what this refers to.

Reply:
I was thinking of storing the latest email alert version for each location in the `info` collection (see https://github.com/covid-projections/covid-projections/pull/2627/files#diff-582f325175286997a5df8ad7cdca7ff4cac8d9f6a17b3100ac4d4ee4169c2278R15), but maybe we can get the latest version from the `vaccination-alerts` collection you proposed here? https://github.com/covid-projections/covid-projections/pull/2627/files#r569154259

Reply:
Ah, cool.  That makes sense.  I was mostly thrown off by "latest updated date" but maybe that was supposed to be "data"?

Also, I think I wouldn't update it until after we've actually emailed users, else I think we lose resumability.